tests/integration/src/test/resources/org/mule/test/integration/locator/component-locator-config.xml

```diff
@@ -14,5 +14,10 @@
         </async>
     </flow>
 
+    <flow name="anotherFlow">
+        <scheduler/>
```

Comment:
Is this valid?  I'm not convinced having default scheduler is a good idea, how does this work?